cli/import-jdl.js

```diff
@@ -204,6 +206,7 @@ const generateEntityFiles = (generator, entity, inFolder, env, shouldTriggerInst
 
             const childProc = forkProcess(runYeomanProcess, [command, ...getOptionAsArgs(options, false, !options.interactive)], { cwd });
             childProc.on('exit', code => {
+                if (code !== 0) setExitCode(code);
```

Comment:
Is the guard required here? Would it be bad to set the code to 0 again? Would make it easier to read imho. And (just a personal matter of taste) in case the if statement is required, I would add `{` like

```json
if (code !== 0) {
 setExitCode(code);
}
```

Reply:
It is, one success will overwrite a fail.
Fork is called at generateApplications, generateEntities and generateDeployments.
Each one, more then one time (ex: this.importState.exportedDeployments.forEach).

An alternative is to exit the script at first fail.

Moved it to the setExitCode.

Reply:
Hi @mshima! nice work :)

If we implement an exit-early strategy at first-fail, we risk making the user unhappy if, for instance, the user encounters more than one error.
Maybe we should go for an hybrid approach:
  - for "light" issues, we can aggregate them
  - for "unrecoverable" ones, I'd rather fail as early as possible

Reply:
I think they are all unrecoverable.
But I suggest create another PR/issue for this.
This PR is kind of minor while fail as early as possible no.

Reply:
Agreed